### DISCUSSION TITLE
A switch which controls vertical or horizontal

### DISCUSSION THREAD

**Discussion by willser** [community/community#72246] [Mobile]
### Select Topic Area

Product Feedback

### Body

Hi,team

This app make me could revoew code and find something intersting anywhere and anytime.Thanks for your contribute.

And I make this discussion for a feature.

Mobile App screen switches between its vertical and horizontal orientations if system auto-rotate turn on.If there is a switch and it controls screen instand of using system-rotate will be good.Because somebody like me not trun auto-rotate on.If I want to review code comfortably,I have to turn on it.It's annoying sometimes.

**Comment by mattruggio** [comment]
Hey @willser, thanks for bringing this use-case to our attention! I created an internal request for our mobile team to investigate this type of application customization. I cannot guarantee any priority on this, as there is already a work-around you suggested, but it seems something worthwhile to consider.
